src/main/java/uk/ac/ic/wlgitbridge/server/Oauth2Filter.java

```diff
@@ -112,4 +112,10 @@ private void unauthorized(HttpServletResponse response) throws IOException {
         unauthorized(response, "Unauthorized");
     }
 
+    private void forbidden(HttpServletRequest request, HttpServletResponse response, String message)
```

Comment:
My understanding is that a "bad password" is usually still a 401, rather than a 403. This discussion seems to back that up: http://stackoverflow.com/questions/3297048/403-forbidden-vs-401-unauthorized-http-responses

Does it need to be a 403 in order to display the message?